ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -196,10 +196,22 @@
           + "whether the Kafka cluster supports the required API, and enables the validator if "
           + "it does.";
 
-  public static final String KSQL_PULL_QUERIES_ENABLE_CONFIG = "ksql.pull.queries.enable";
-  public static final String KSQL_PULL_QUERIES_ENABLE_DOC =
+  public static final String KSQL_QUERY_PULL_ENABLE_CONFIG = "ksql.query.pull.enable";
+  public static final String KSQL_QUERY_PULL_ENABLE_DOC =
       "Config to enable or disable transient pull queries on a specific KSQL server.";
-  public static final boolean KSQL_PULL_QUERIES_ENABLE_DEFAULT = true;
+  public static final boolean KSQL_QUERY_PULL_ENABLE_DEFAULT = true;
+
+  public static final String KSQL_QUERY_PULL_OWNER_TIMEOUT_MS_CONFIG =
```

Comment:
Naming the method on owner etc internally is okay for now. But, I feel for a public config, we can better standardize the terminology.. 

IMO owner_timeout is not very intuitive.. Can we please name this `ksql.query.pull.routing.timeout.ms`? That's what we are trying to achieve by finding the owner information..

Reply:
Sure, will change the name.